app/views/layouts/application.html.erb

```diff
@@ -137,6 +137,7 @@
               <% end %>
               <%= menu_item "<i class='fa fa-cog'></i> Settings".html_safe, edit_volunteer_path(current_volunteer.id) %>
               <%= menu_item "<i class='fa fa-file-o'></i> Waiver".html_safe, new_waiver_url %>
+              <%= menu_item "<i class='fa fa-file-o'></i> Driver Waiver".html_safe, "/waiver/driver-new" %>
```

Comment:
Please replace the `"/waiver/driver-new"` with a rails helper like the ones that are used in lines 138 and 139.